andela/team-odd-bn-frontend
# #169258599 Travel Admin can create accommodation facility

#### What does this PR do?
This PR adds functionality for a travel admin to create an accommodation facility for users
#### Description of Task to be completed?
- The travel admin is able to enter a name for the location/accomodation facility
- The travel admin is able to enter the location (I.E address) of the center
- The travel admin is able to upload an image of the centre 
- The travel admin is able to create a number of rooms with room names and room type
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-frontend`
- `git checkout ft-create-accommodation-facility-169258599`
- `npm install` packages
- `npm run dev` to start the development server
- It should open your default browser automatically
- Navigate to `localhost:4000/accommodation/create` to access the page directly
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169258599](https://www.pivotaltracker.com/story/show/169258599)
#### Screenshots (if appropriate)
###### Adding photos
<img width="1290" alt="Screen Shot 2020-01-16 at 11 30 54" src="https://user-images.githubusercontent.com/35301129/72512359-9ff0fb00-3854-11ea-918a-57bd244ac3d4.png">


###### Adding rooms
<img width="1440" alt="Screen Shot 2020-01-20 at 15 30 30" src="https://user-images.githubusercontent.com/35301129/72730944-43b31180-3b9b-11ea-888b-937800bf25f9.png">

###### creating accommodation
<img width="1436" alt="Screen Shot 2020-01-20 at 12 51 50" src="https://user-images.githubusercontent.com/35301129/72720972-b6b08e00-3b83-11ea-8eac-916eb1730008.png">


#### Questions:
N/A
